projects/chaos/src/power_path.c

```diff
@@ -4,14 +4,23 @@
 #include <string.h>
 
 #include "adc.h"
+#include "can_msg_defs.h"
+#include "can_transmit.h"
 #include "chaos_events.h"
 #include "event_queue.h"
 #include "gpio.h"
 #include "gpio_it.h"
 #include "interrupt.h"
+#include "log.h"
 #include "soft_timer.h"
 #include "status.h"
 
+// All values in millivolts
+#define POWER_PATH_AUX_UV 8460
```

Comment:
Might as well add the millivolt unit in the name

Reply:
Done.